### PR TITLE
add check if dss redirect fails

### PR DESCRIPTION
**Pull request by ann-sherin**
Currently, DSS redirect command doesn't raise exception if it fails. So, check added to raise exception when dss redirect run fails in upgrades code.

### CODE REVIEW

File: PyDSS/utils/utils.py [hunk 1]
```diff
@@ -20,6 +20,12 @@
 logger = logging.getLogger(__name__)
 
 
+def check_redirect(file_name):
```
Comment (daniel-thom) [inline PyDSS/utils/utils.py]: Would you mind making a proper docstring?  You can use the other functions in this file as examples.  For the exception you would add something like this:

```
"""
Raises
-------
Exception
    Raised if the command fails.

"""
```

File: PyDSS/utils/utils.py [hunk 2]
```diff
@@ -20,6 +20,12 @@
...
+    result = dss.run_command("Redirect {}".format(file_name))
+    if result != '':
+        raise Exception(f'Redirect failed {result}')
```
Comment (daniel-thom) [inline PyDSS/utils/utils.py]: Can you include the filename in the exception string?  OpenDSS may not report anything about it if failure happens in a secondary redirect.

File: PyDSS/utils/utils.py [hunk 3]
```diff
@@ -20,6 +20,12 @@
 logger = logging.getLogger(__name__)
 
 
+def check_redirect(file_name):
+    result = dss.run_command("Redirect {}".format(file_name))
```
Comment (daniel-thom) [inline PyDSS/utils/utils.py]: Can you include a debug log message just before running it?

```
logger.debug(...)
```

Comment (daniel-thom) [inline PyDSS/utils/utils.py]: This isn't a request for a change, but I do suggest that you try to be consistent with your style.  f-strings vs `"{}".format()`, single quotes vs double quotes.  :)

File: PyDSS/utils/utils.py (hunk 1 above)
Comment (ann-sherin) [reply PyDSS/utils/utils.py]: Okay, I'll add that

**Pull request merged**